### ISSUE TITLE
Lol wont save game settings

### ISSUE THREAD

**Issue by RigobertoEHA1** [iholston/lol-bot#270]
ive reinstall macos sequoia from 0, but when i download and play league of legends the game.cfg doest appear, ive try reinstall lol, i used the hextech tool, and configs always reset every game, the file config.cfg dont appear, for this reason the bot cant run on full screen and 1080p, iveen change the permission of the root of the game to read and write, but not resolve that

**Comment by RigobertoEHA1** [comment]
the bot create the file config.cfg on the game but the game dont take it changes, please help me

**Comment by iholston** [comment]
Can you add a copy of the game.cfg that the bot is using to this thread?

**Comment by iholston** [comment]
And are you saying that the window opens as 1080p window? In the mean time you could simply set the client to the lowest resolution borderless and it should work.

**Comment by RigobertoEHA1** [comment]
[General]
WindowMode = 2
Height = 768
Width = 1024

[Performance]
ShadowQuality = 0
FrameCapType = 5
EnvironmentQuality = 0
EffectsQuality = 0
CharacterQuality = 0
EnableGrassSwaying = 0
EnableFXAA = 0


is my game.cfg, the config every game reboot to the default, the game no create a file, and I've change the bot.py
config_dir = os.path.join(self.config['macos_install_dir'], 'Contents/LoL/Config')
because the dir starts with capital letters like that

**Comment by iholston** [comment]
That is the correct game file. Is the game window not opening as a 1024x768 borderless?

**Comment by RigobertoEHA1** [comment]
no, not open, and dont save the config, every match need change manually

**Comment by iholston** [comment]
Okay, thanks, I'll look into it.

**Comment by iholston** [comment]
I'm sorry, to clarify, you are saying that when you run the bot the game settings reset to default every game? What resolution is the window that is opening. Is it borderless? I can't seem to recreate this issue myself. You're game.cfg file is the correct one (it will look a little different after the first game because league of legends adds a bunch of information). Maybe try deleting your persistedsettings.json file?

**Comment by RigobertoEHA1** [comment]
maybe its for the macos sequoia, when i install the game need rename League Of Legends for LeagueOfLegends, if i not do this, the ingame cant launch, maybe this currupt the game, and wonts save the changes, the game run on fullscreen 1080p

**Comment by iholston** [comment]
I'm taking a look at this issue again, you are saying that no config.cfg file is created? The file should be deleted and then recreated with the correct settings. Additionally, the persisted settings.json file should be overwritten and set to read only before the bot runs. Can you add logs for the bot here?
